orchestrator/services/audit_retention.py

```
@@ -0,0 +1,140 @@
+                workspace_id=str(workspace_id),
+                user_id=None,
+                actor_type="system",
+                action="audit:retention_sweep",
+                resource_type="audit_logs",
+                details={
+                    "cutoff": cutoff.isoformat(),
+                    "rows_deleted": int(cnt),
+                    "retention_days": retention_days,
+                },
+            )
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)